Code/Editor/TrackView/SequenceBatchRenderDialog.cpp

```diff
@@ -906,43 +897,30 @@ void CSequenceBatchRenderDialog::CaptureItemStart()
 
     CV_TrackViewRenderOutputCapturing = 1;
 
-    SRenderItem renderItem = m_renderItems[m_renderContext.currentItemIndex];
-    IAnimSequence* nextSequence = renderItem.pSequence;
-    // Initialize the next one for the batch rendering.
-    // Set the active shot.
-    m_renderContext.pActiveDirectorBU = nextSequence->GetActiveDirector();
-    nextSequence->SetActiveDirector(renderItem.pDirectorNode);
-
-    // Back up flags and range of the sequence.
-    m_renderContext.flagBU = nextSequence->GetFlags();
-    m_renderContext.rangeBU = nextSequence->GetTimeRange();
-
-    // Change flags and range of the sequence so that it automatically starts
-    // once the game mode kicks in with the specified range.
-    nextSequence->SetFlags(m_renderContext.flagBU | IAnimSequence::eSeqFlags_PlayOnReset);
+    const SRenderItem& renderItem = m_renderItems[m_renderContext.currentItemIndex];
 
     m_renderContext.captureOptions.timeStep = 1.0f / renderItem.fps;
-
-    Range newRange = renderItem.frameRange;
-    newRange.start -= m_renderContext.captureOptions.timeStep;
-    renderItem.pSequence->SetTimeRange(newRange);
-
+  
     // Set up the custom config cvars for this item.
-    for (size_t i = 0; i < renderItem.cvars.size(); ++i)
+    IConsole* pConsole = GetIEditor()->GetSystem()->GetIConsole();
```

Comment:
Same here, this is the legacy CryEngine console, now superseeded by the AzCore IConsole. You can use it via AZ::Interface<AZ::IConsole>::Get()->PerformCommand()